ui/core-react/src/core-react/dialog/Dialog.tsx

```diff
@@ -143,36 +140,15 @@ export class Dialog extends React.Component<DialogProps, DialogState> {
     trapFocus: false,
   };
 
-  /** @internal */
-  public override readonly state: Readonly<DialogState>;
-
   constructor(props: DialogProps) {
     super(props);
-    this.state = {
-      rightResizing: false,
-      downResizing: false,
-    };
-  }
-
-  private getParentWindow() {
-    // istanbul ignore next
-    return this._parentDocument.defaultView ?? window;
   }
 
   public override componentWillUnmount(): void {
-    const parentWindow = this.getParentWindow();
```

Comment:
Have you checked if dialog resizing continues to work in a popout window?

Reply:
Works.